### PR TITLE
Networking Sandbox Blueprint

### PR DESCRIPTION
**Pull request by sruffilli**
This blueprint creates a networking playground showing a number of different VPC connectivity options:

* Hub and spoke via HA VPN
* Hub and spoke via VPC peering
* Interconnecting two networks via a network virtual appliance (aka NVA)

On top of that, this blueprint implements Policy Based Routing (aka PBR) to show how to force all traffic within a VPC to be funneled through an internal network passthrough load balancer, to implement an Intrusion Prevention System (IPS). PBR is enabled in the `hub` VPC, matching all traffic originating from within that VPC.

---
**Checklist**

I applicable, I acknowledge that I have:
- [x] Read the [contributing guide](https://github.com/GoogleCloudPlatform/cloud-foundation-fabric/blob/master/CONTRIBUTING.md)
- [x] Ran `terraform fmt` on all modified files
- [x] Regenerated the relevant README.md files using [`tools/tfdoc.py`](https://github.com/GoogleCloudPlatform/cloud-foundation-fabric/blob/master/CONTRIBUTING.md#fabric-tools)
- [x] Made sure all relevant tests pass

### CODE REVIEW

**Comment by ludoo** [comment]
I would change the name to be more specific :)

**Comment by sruffilli** [comment]
> I would change the name to be more specific :)

Open to suggestions!

**Comment by ludoo** [comment]
`vpc-connectivity-lab`?

File: blueprints/networking/vpc-connectivity-lab/variables.tf [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+  default = {
+    ext         = "10.255.0.0/16"
+    hub-a       = "10.0.1.0/24"
+    hub-all     = "10.0.0.0/16"
+    hub-b       = "10.0.2.0/24"
+    hub-nva     = "10.0.0.0/24"
+    int         = "10.0.0.0/9"
+    peering-a   = "10.3.0.0/24"
+    peering-b   = "10.4.0.0/24"
+    rfc1918_10  = "10.0.0.0/8"
```
Comment (ludoo) [inline blueprints/networking/vpc-connectivity-lab/variables.tf:29]: nit: I would hardwire these

**Review by ludoo — APPROVED** [review]

File: blueprints/networking/vpc-connectivity-lab/variables.tf (hunk 1 above)
Comment (sruffilli) [reply blueprints/networking/vpc-connectivity-lab/variables.tf:29]: That's where I came from, but other blueprints use the same data structure, and I don't dislike it.

**Pull request merged**